src/solver.ts

```diff
@@ -0,0 +1,57 @@
+
+
+function solver(input: string):string
+{
+    let nums = input.split(' ');
+    let entries :Entry[] = [];
+
+    for(let i = 0; i < nums.length; i++)
+    {
+        let num = nums[i];
+        let weight = 0;
+
+        for(var j = 0; j < num.length; j++)
+        {
+            let digit = parseInt(num[j]);
+            weight += digit;
+        }
+
+
+        entries.push( new Entry(num, weight));
```

Comment:
Creating a new instance of 'Entry' in every push is causing performance issues.
btw, by reducing the amount of for loops will decrement the execution time